src/pages/wallets/Wallet.js

```diff
@@ -112,15 +121,60 @@ function Wallet(props) {
       dataField: "collateral",
       text: "collateral",
       sort: true,
-      formatter: (cell, row) => <Value value={cell} decimals={2} compact />,
+      formatExtraData: { timePeriod },
+      formatter: (cell, row) => (
+        <>
+          <div className="text-nowrap">
+            <Value value={cell} decimals={2} compact />
+            <br />
+            <ValueChange
+              className="pl-2"
+              value={
```

Comment:
does the ```value={row[`collateral_change_${timePeriod}d`}``` not work?

Reply:
No, it does not, it returns nothing
it wants a dot after the row, and then it raises errors, stating that row is not a function
value={row.[`collateral_change_${timePeriod}d`} / value = {row.collateral_change_d30}
If I put the dot inside the [ ], it again returns nothing.